Doc-11534-PM-2.8-Edits-DH  Partner Manager 2.8 edits (#141)
mulesoft/docs-partner-manager: modules/ROOT/Edits/endpoints.adoc

= Endpoints

> Endpoints define the transport protocol configurations necessary to receive or send messages with external trading partners or internal backend applications.

Reviewer: "to receive or send messages with" is not grammatical (s/b "receive from", "send to"). This is much simpler:
```suggestion
Endpoints define the transport protocol configurations necessary to communicate with external trading partners or internal backend applications.
```
```

Partner Manager uses endpoints to receive transmissions from partners and messages from backend application, define transport protocols, and specify message or message acknowledgment destinations.

== Endpoint Ownership

> You can assign the host, partner, or 3rd party connection as the owner of endpoints to control the use of endpoints within inbound and outbound message flows.

Reviewer: Is this the same?
```suggestion
You can assign the host, partner, or 3rd party connection as the owner [[and controller?]] of the endpoints in inbound and outbound message flows.
```

> When creating partner-facing endpoints directly from the partner, or 3rd party connecction, the ownership of the endpoint is automatically assigned to the entity from which the endpoint was created.

Reviewer: Lack of agreement between intro. modifier and subject of sentence

In a message flow, availability depends on the endpoint assignment:

* Any partner can select an endpoint assigned to the host.

> * If an endpoint is assigned to a specific partner or 3rd party connection, than that endpoint is available only to the partner for which the endpoint is assigned.

Reviewer: ```suggestion
* If an endpoint is assigned to a specific partner or 3rd party connection, then that endpoint is available only to that assigned entity.
```
